android/src/main/java/com/media4care/twilio/video/TwilioVideoActivity.java

```diff
@@ -484,11 +488,16 @@ public void onConnected(Room room) {
             @Override
             public void onReconnecting(@NonNull Room room, @NonNull TwilioException twilioException) {
                 sendTwilioEvent(TWILIO_EVENT_RECONNECTING);
+
+                statusText.setVisibility(View.VISIBLE);
+                statusText.setText(R.string.reconnecting);
```

Comment:
For us, it doesn't matter, but for library users, I think it's better to add all strings as plugin parameters. 

1. Otherwise, they have to change the plugin itself to add a new language.
2. It keeps all internationalized strings nicely in the same place (for us, in en.json / de.json)